integration/kind-test/scripts/run-tests.sh

```diff
@@ -40,13 +44,24 @@ if [ "$exit_code" -eq 0 ] ; then
     exit $?
   fi
 
-  go run $SCENARIOS/runner/main.go $NAMESPACE $SERVICE $CLUSTERID1 $CLUSTERSETID1 $ENDPT_PORT $SERVICE_PORT "$updated_endpoints"
+  go run $SCENARIOS/runner/main.go $NAMESPACE $SERVICE $CLUSTERID1 $CLUSTERSETID1 $ENDPT_PORT $SERVICE_PORT $SERVICE_TYPE "$updated_endpoints"
   exit_code=$?
 
   if [ "$exit_code" -eq 0 ] ; then
     ./integration/shared/scripts/test-import.sh "$UPDATED_ENDPOINT_COUNT" "$updated_endpoints"
     exit_code=$?
   fi
+
+  if [ "$exit_code" -eq 0 ] ; then
+    ./integration/kind-test/scripts/DNS-test.sh
+    exit_code=$?
+  fi
+fi
+
+# Scale deployment back down for future test and delete service export
+if [ "$exit_code" -eq 0 ] ; then
+  $KUBECTL_BIN scale deployment/"$deployment" --replicas="$EXPECTED_ENDPOINT_COUNT" --namespace "$NAMESPACE"
+  $KUBECTL_BIN delete ServiceExport $SERVICE -n $NAMESPACE
 fi
```

Comment:
We should probably sleep a few seconds to let the controller clean up before killing it after deleting the export.